### ISSUE TITLE
type decimal length/values 2nd can't be set

### ISSUE THREAD

**Issue by hipkiss91** [phpmyadmin/phpmyadmin#13537]
### Steps to reproduce
1. create new table.
2. new column with type decimal and type "6,4" into the length/values field
3. click save. the column defaults to (10,0)

### Expected behaviour
should be (6,4) not (10,0)

Got version 5.7.14 MYSQL, running on win10 64 bit through WAMP 3.0.6. 
I can create a column of decimal and change the first number but can't change the second?

**Comment by ibennetch** [comment]
I can't reproduce on the demo server with phpMyAdmin 4.7.4 or master, can you reproduce it there (https://demo.phpmyadmin.net)?

Which phpMyAdmin version are you using? Can you create a table from the command line client to verify whether it works as expected there? The SQL with an already-existing database 'test' might look like:

```
CREATE TABLE `test`.`dectest` ( `d` DECIMAL(6,4) NOT NULL ) ENGINE = InnoDB;
```

**Comment by nijel** [comment]
If you're using older version, see #12480

**Comment by hipkiss91** [comment]
I'm rocking v4.6.4 phpmyadmin.
It happens when I use the GUI. However, I found that editing inline after creating the column worked.

**Comment by nijel** [comment]
Okay, please upgrade to more recent version, this has been fixed in 4.6.5.
